indy_common/authorize/auth_map.py

```diff
@@ -68,6 +68,16 @@
                              old_value='*',
                              new_value='*')
 
+add_context = AuthActionAdd(txn_type=SET_CONTEXT,
```

Comment:
Please extend `https://github.com/hyperledger/indy-node/blob/master/docs/source/auth_rules.md` for new transactions

Reply:
added in [655aed4](https://github.com/hyperledger/indy-node/pull/1430/commits/655aed4f4f224b429304d343b0c9c69737635bf8)